resources/infrastructure_logger.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<logging>
+    <loggerOptions>
+        <loggerOption LoggerName="FileLogger"   >false</loggerOption>
+        <loggerOption LoggerName="ConsoleLogger">false</loggerOption>
+    </loggerOptions>
+    <logLevels>
+        <logLevel level="ERROR"  >false</logLevel>
+        <logLevel level="WARNING">false</logLevel>
+        <logLevel level="SUCCESS">false</logLevel>
+        <logLevel level="INFO"   >false</logLevel>
+    </logLevels>
+</logging>
```

Comment:
Add the test mode option and a path to the other config file

Reply:
should I make a separate PR for that?

Reply:
No, edit here itself